string nodeName
string packageName
string[] runningNodeList
string launchFileName
string launchParams
---
bool res
